package.json

```diff
@@ -8533,10 +8533,10 @@
                                     },
                                     {
                                         "key": "M",
-                                        "name": "Maximize window and hide side bar",
+                                        "name": "Maximize window without hiding others",
```

Comment:
Since now we have actual maximize window now, should we call this what it is "Toggle window width"? @MarcoIeni